compiler/noirc_frontend/src/hir/def_collector/dc_crate.rs

```diff
@@ -388,20 +389,14 @@ impl DefCollector {
                         let result = current_def_map.modules[resolved_import.module_scope.0]
                             .import(name.clone(), visibility, module_def_id, is_prelude);
 
-                        // Empty spans could come from implicitly injected imports, and we don't want to track those
-                        if visibility != ItemVisibility::Public
-                            && name.span().start() < name.span().end()
-                        {
```

Comment:
Why was this check removed?

Reply:
It was moved to `UsageTracker::add_unused_item`